controllers/nstemplateset/cluster_resources_test.go

```diff
@@ -103,12 +106,24 @@ func TestClusterResourceKinds(t *testing.T) {
 			}
 
 			// when
-			existingResources, err := clusterResourceKind.listExistingResources(fakeClient, "johny")
+			existingResources, err := clusterResourceKind.listExistingResources(fakeClient, "johny", apiGroups)
 
 			// then
 			require.Error(t, err)
 			require.Len(t, existingResources, 0)
 		})
+
+		t.Run("listExistingResources should return not return any resource when APIGroup is missin for gvk "+clusterResourceKind.gvk.String(), func(t *testing.T) {
```

Comment:
```suggestion
		t.Run("listExistingResources should return not return any resource when APIGroup is missing for gvk "+clusterResourceKind.gvk.String(), func(t *testing.T) {
```

Reply:
fixed as part of https://github.com/codeready-toolchain/member-operator/commit/74ff7e3d8aa087a70c8b6a8b3863eac8d4ff2642